### PR TITLE
Float button for pages selector

### PR DESCRIPTION
**Pull request by Rikahei**
screenshot 2018-06-14 04 45 27

Hi, 
I made a float button for pages selector based on original code.

### CODE REVIEW

**Comment by hk-ng** [comment]
Relate to #9

File: client/web/src/containers/Topics/Thread/index.jsx [hunk 1]
```diff
@@ -100,6 +100,14 @@ const styles = theme => ({
...
     marginRight: `${theme.margin}px !important`,
   },
+  floatButton: {
+    background: theme.primaryColor,
+    padding: '1px',
+    borderRadius: '12px',
+    position: 'fixed',
+    bottom: '210px',
+    right: '4%',
+  }
```
Comment (ckwun) [inline client/web/src/containers/Topics/Thread/index.jsx]: Add trailing comma

File: client/web/src/containers/Topics/Thread/index.jsx [hunk 2]
```diff
@@ -257,4 +277,4 @@ const enhance = compose(
   pure,
 );
 
-export default enhance(Thread);
+export default enhance(Thread);
```
Comment (ckwun) [inline client/web/src/containers/Topics/Thread/index.jsx]: Add newline at end of file

File: client/web/src/containers/Topics/Thread/index.jsx [hunk 3]
```diff
@@ -167,6 +175,18 @@ const Thread = ({
...
+      <Button disabled={page === 1} type="primary" onClick={() => { handlePageChange(page - 1); }}>
+        <Icon type="left" />
+      </Button>
+      <Select value={page} className={classes.floatSelect} showArrow={false} 
+      onChange={handlePageChange} dropdownMatchSelectWidth={false} style={{ width: 80 }}>
+        {pageOptions}
+      </Select>
+      <Button disabled={page === totalPage} type="primary" onClick={() => { handlePageChange(page + 1); }}>
+        <Icon type="right" />
+      </Button>
```
Comment (ckwun) [inline client/web/src/containers/Topics/Thread/index.jsx]: Remove trailing space

File: client/web/src/containers/Topics/Thread/index.jsx [hunk 4]
```diff
@@ -167,6 +175,18 @@ const Thread = ({
         下一頁<Icon type="right" />
       </Button>
     </div>
+    <div className={classes.floatButton}>
+      <Button disabled={page === 1} type="primary" onClick={() => { handlePageChange(page - 1); }}>
+        <Icon type="left" />
+      </Button>
+      <Select value={page} className={classes.floatSelect} showArrow={false}
```
Comment (ckwun) [inline client/web/src/containers/Topics/Thread/index.jsx]: Break props to new line and remove trailing space
```
<Select
  value={page}
  className={classes.floatSelect}
  showArrow={false}
  onChange={handlePageChange}
  dropdownMatchSelectWidth={false}
  style={{ width: 80 }}
>
```

**Comment by hk-ng** [comment]
Is there anybody think that floating action button with a drop down list is a bad ux?

**Comment by ckwun** [comment]
Enhanced and added by #14, thanks for the contribution

**Pull request closed without merging**